src/phovea.ts

```diff
@@ -99,6 +100,12 @@ export default function (registry: IRegistry) {
     priority: 30
   });
 
+  registry.push(EP_ORDINO_LOGO, 'ordino_logo', () => ({}), <IOrdinoLogoDesc>{
+    loadIcon() {
+      return import('ordino/dist/assets/logos/ordino.svg');
+    },
```

Comment:
```suggestion
  registry.push(EP_ORDINO_LOGO, 'ordino_logo', function () {
    return import('ordino/dist/assets/logos/ordino.svg').then(PluginRegistry.getInstance().asResource), <IOrdinoLogoDesc>{
```

Could we use the loading mechanism similar to the locales (see the next extension point below) instead?